api/models.py

```diff
@@ -228,3 +229,54 @@ class Tag(ModelReprMixin, models.Model):
 
     def __str__(self):
         return self.title
+
+
+class Infraction(ModelReprMixin, models.Model):
+    """An infraction for a member of our Discord server."""
+
+    TYPE_CHOICES = (
+        ("note", "Note"),
+        ("warning", "Warning"),
+        ("mute", "Mute"),
+        ("ban", "Ban"),
+        ("kick", "Kick"),
+        ("superstar", "Superstar")
+    )
+    id = models.UUIDField(
+        primary_key=True,
+        default=uuid.uuid4,
+        help_text="The UUID of the infraction."
+    )
```

Comment:
You can omit this and Django will generate an ID field. Why UUIDs, anyways?

Reply:
The comment for the old schema said it's a UUID.

Reply:
Well, but that's NoSQL... I don't think using a regular ID would hurt here.